src/index.js

```diff
@@ -1,31 +1,71 @@
-// import _ from 'lodash';
+import { findCity, findWeather } from './resource';
 
+const searchForm = document.querySelector('form');
+const displayCard = document.querySelector('.display-card');
+const weatherDetails = document.querySelector('.weather-details');
+const time = document.querySelector('img.time');
+const iconImg = document.querySelector('.icon img');
 
-// function component() {
-//   const element = document.createElement("div");
+const addCity = async (city) => {
+  const cityKey = await findCity(city);
+  const cityWeather = await findWeather(cityKey.Key);
 
-//   // Lodash, currently included via a script, is required for this line to work
-//   element.innerHTML = _.join(["Hello", "webpack"], " ");
+  return {
+    cityKey,
+    cityWeather,
+  };
+};
+
+const celciusToFarenheitEvent = () => {
+  const farenheitBtn = document.querySelector('.farenheitBtn');
+  const farenheit = document.querySelector('.farenheit');
+  farenheitBtn.onclick = () => {
+    if (farenheit.style.display === 'none') {
+      farenheit.style.display = 'block';
+    } else {
+      farenheit.style.display = 'none';
+    }
+  };
+};
 
-//   return element;
-// }
+const display = (info) => {
+  const { cityKey } = info;
+  const { cityWeather } = info;
 
-// document.body.appendChild(component());
+  weatherDetails.innerHTML = `
+                   <h5 class="location">${cityKey.EnglishName}</h5>
+                   <div>${cityWeather.WeatherText}</div>
+                   <div id="cel-container" class=" my-4">
+                        <span class="celcius">${cityWeather.Temperature.Metric.Value} &deg;C</span></br>
+                        <button class="farenheitBtn">Farenheit</button>
```

Comment:
You should use a toggle control to determine if the result will be shown in Celsius or Fahrenheit ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))

Kindly make sure your toggle implementation is working fine.